docs/tutorial/bit_extraction.md

```diff
@@ -0,0 +1,122 @@
+# Bit Extraction
+
+Some applications require directly manipulating bits of integers. Concrete provides bit extraction operation for such applications.
+
+{% hint style="warning" %}
+Bit extraction only works in `Native` encoding, which is usually selected when all table lookups in the circuit are below or equal to 8 bits.
+{% endhint %}
+
+```python
+from concrete import fhe
+
+@fhe.compiler({"x": "encrypted"})
+def f(x):
+    return fhe.bits(x)[0], fhe.bits(x)[3]
+
+inputset = range(32)
+circuit = f.compile(inputset)
+
+assert circuit.encrypt_run_decrypt(0b_00000) == (0, 0)
```

Comment:
Otherwise examples are perfectly clear thx to 0b_.

Reply:
Since it's a single bit extraction (e.g. `fhe.bits(x)[0]`), I think it's better to use `0` or `1` to represent the bit.